src/main/rx/fport.c

```diff
@@ -245,18 +244,6 @@ static void smartPortWriteFrameFport(const smartPortPayload_t *payload)
 }
 #endif
 
-static bool checkChecksum(uint8_t *data, uint8_t length)
-{
-    uint16_t checksum = 0;
-    for (unsigned i = 0; i < length; i++) {
-        checksum = checksum + *(uint8_t *)(data + i);
-    }
-
-    checksum = (checksum & 0xff) + (checksum >> 8);
```

Comment:
All you have to do is to apply this line twice - checksum may be over 0xff after first sum (0xff + some overflow value)

Reply:
Indeed, I didn't want to introduce any new bug, I just took the official algo